.github/workflows/linux_container_tests.yml

```diff
@@ -78,77 +116,21 @@ jobs:
         uses: actions/setup-dotnet@67a3573c9a986a3f9c594539f4ab511d57bb3ce9 # v4.3.1
         with:
           dotnet-version: '9.0.x'
-  
```

Comment:
It's a little hard to follow the changes here, but what's happening is that 2 separate jobs (amd64 / arm64) are now running as a single matrix job. So all these steps that look like they were dumped are just the steps from one of the 2 jobs that got coalesced into a single job.